src/components/Form/TextInput/TextInput.js

```diff
@@ -65,21 +65,52 @@ TextInput.defaultProps = {
 };
 
 TextInput.propTypes = {
+
+    /** If true, user won't be able to interact with the component.*/
     disabled: PropTypes.bool,
+
+    /** Bind the error message's object in the prop that will be executed when the validation failed.
+     * Whether to style the TextInput with error style.*/
+
```

Comment:
Remove `errors` description in the meantime as current behavior for this will be deprecated.